R/impute_daily_incidence.R

```diff
@@ -0,0 +1,23 @@
+#' Imputed incidence value for the single day with reporting anomaly
+#'
+#' @param data time series data for one location in adjustment case
+#' @param adjustment_case a case to adjust
+#' @param measure case or death
```

Comment:
Let's clarify this documentation a little.  Note that I changed the pluralization here.  If we pluralize, will we break any code?

```suggestion
#' @param measure String specifying measure: "cases" or "deaths"
```